s32/mcux/devices/S32K344/S32K344_features.h

```diff
@@ -7,7 +7,7 @@
 #ifndef _S32K344_FEATURES_H_
 #define _S32K344_FEATURES_H_
 
-#include "autoconf.h"
+#include <zephyr/autoconf.h>
```

Comment:
Will this break CI until the zephyr side merges?

Reply:
#63973 has been merged last week